floor_generator/lib/writer/query_method_writer.dart

```diff
@@ -116,7 +116,7 @@ class QueryMethodWriter implements Writer {
 
   @nonNull
   String _generateNoReturnQuery(@nullable final String arguments) {
-    final parameters = StringBuffer()..write("'${_queryMethod.query}'");
+    final parameters = StringBuffer()..write('r""" ${_queryMethod.query} """');
```

Comment:
❓ Did you add the leading and trailing space on purpose?
```suggestion
    final parameters = StringBuffer()..write('r"""${_queryMethod.query}"""');
```

Reply:
Yes, if the query contains a `""` at the end then it will be concatenated to `"""""`, which dart then will interpret as `""" ""`, which is wrong. so I put a space at the front (shouldn't matter, just in case) and a space at the end. Both spaces should be permitted by sqlite's syntax.

Example query (although `"`s there are not really valid sql, but we shouldn't change them anyways.):
```sql
SELECT * FROM table WHERE x=""
```
(`"` quotes in sqlite are used for identifiers https://www.sqlite.org/lang_keywords.html)
I added a test case for the trailing `'` quotes before but forgot to add one for the other ones. I'll be adding another one for the `"` quotes now. I think I'll also add one for backtick (` ` ` ) quotes, even if they're not causing trouble right now.